test/uk/gov/hmrc/transitmovementsconverter/models/ModelImplicitsSpec.scala

```diff
@@ -83,7 +87,7 @@ class ModelImplicitsSpec extends AnyFreeSpec with ScalaFutures with Matchers wit
     }
 
     "A phase ID of 'NCTS5.1' should be written for NCTS5u461" in {
-      ModelImplicits.phaseIDtypeWrites.writes(NCTS5u461) mustBe JsString("NCTS5.1")
+      ModelImplicits.phaseIDtypeWrites.writes(NCTS5u461) mustBe JsString(" NCTS5.1")
```

Comment:
This too.

Reply:
updated stypes.xsd to remove empty space